src/features/virtualDocumentTracker.ts

```diff
@@ -50,7 +50,7 @@ function trackFutureVirtualDocuments(server: OmniSharpServer, eventStream: Event
 }
 
 function shouldIgnoreDocument(document: TextDocument, server: OmniSharpServer): boolean {
-    if (document.uri.scheme === 'file' || document.languageId !== 'csharp') {
+    if (document.uri.scheme.startsWith("virtualCSharp-") || document.languageId !== 'csharp') {
```

Comment:
Need to also ensure that `uri.scheme !== "virtualCSharp"`. Don't want to encourage bad behavior 😄

Reply:
Wouldn't the dash at the end "virtualCSharp-" take care of that?

Reply:
Mistyped, need to make sure people don't bind to `virtualCSharp-`